src/cljs/athens/events.cljs

```diff
@@ -1884,23 +1884,23 @@
 
 (reg-event-fx
   :unlinked-references/link
-  (fn [_ [_ {:block/keys [string uid] :as block} title]]
+  (fn [_ [_ {:block/keys [string uid]} title]]
     (js/console.debug ":unlinked-references/link:" uid)
-    (if-let [local? (not (client/open?))]
```

Comment:
You've negated here previous `if` condition, but `then` & `else` parts not.

Reply:
Updated